Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```diff
@@ -72,73 +83,43 @@
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSecurity
-  name: security over-ear headset
+  name: security command over-ear headset
```

Comment:
unnecessary name change

Reply:
Necessary, else it is common headset for whole security